examples/filesystem/README.md

```diff
@@ -1,31 +1,39 @@
 # File system usage example
 
-This is a basic example how to use a file system with RIOT in your embedded application.
+This basic example shows how to use a file system with RIOT in your embedded
+application.
 
-This example shows:
- - how to mount/format/unmount a filesystem, with spiffs, littlefs and constfs examples
- - how to open/read/write/close a file with and without newlib
+In particular, this example shows:
+
+- how to mount/format/unmount a filesystem, with spiffs, littlefs and constfs
+  examples
+- how to open/read/write/close a file with and without newlib
 
 In RIOT, most filesystems use `mtd` as flash interface. So to use this example
-you must define `MTD_0`. `MTD_0` is a pointer to a `mtd_dev_t` instance.
-This example use `littlefs` as default filesystem on the whole `mtd` device.
+one must define `MTD_0`. `MTD_0` is a pointer to a `mtd_dev_t` instance.
+
+This example uses `littlefs` as default filesystem on the whole `mtd` device.
 A `constfs` filesystem is also demonstrated with two files.
 
-All the RIOT filesystems are used through the `vfs` interface, and on most platform
-they can be accessed transparently with `open/close/read/write/...` functions.
+All the RIOT filesystems are used through the `vfs` interface, and on most
+platforms they can be accessed transparently with `open/close/read/write/...`
+functions.
```

Comment:
Again there is this "most platforms". I'd like to have a list at least for those which are not supported. Moreover, it's confusing the way it's written `they can be accessed transparently`, who's 'they'? If it's the file system, I'd say is wrong to say you can access it transparently using those commands, but you rather access files. If it's not the file system, then you need to specify that you can actually access 'files'.